src/test/scala/com.snowplowanalytics.snowplow.micro/events.scala

```diff
@@ -89,7 +89,7 @@ object events {
 
     RawEvent(
       collectorApi,
-      params,
+      params.map({ case (k, v) => (k, Option(v)) }),
```

Comment:
That will work.

For the benefit of your scala education 😉 it is unusual to see `Option(...)` used in this way, unless you are wrapping values which you suspect might be null, e.g. those returned from a java library.  In this unit test you are in control of the params, so you know they are not null.  Therefore I would expect to see `Some` instead of `Option`:
```scala
params.map({ case (k, v) => (k, Some(v)) }),
```

or this might also compile if you are lucky:
```scala
params.mapValues(Some(_))
```

Reply:
Thanks for the tip!

Reply:
Just be careful with `mapValues` in pre-2.13 as its lazy https://gist.github.com/davidandrzej/5006950 and side-effects there will be dangerous